lib/ItemRow/README.md

```diff
@@ -1,22 +1,32 @@
 # ItemRow
-A component that renders a row for content Items showing the title, status and a label (if required).
+A compound component that renders a row for content Items.
 
 ## Usage
 
 ### Props
 
 | Name                | Type          | Default   | Required | Description                                                                            |
 | ------------------- |-------------- | --------- | -------- |--------------------------------------------------------------------------------------- |
-| children            | Node          | N/A       | Yes      | A node which renders inside a row. Mostly likely a text and a link.                    |
-| indicator           | Node          | null      | No       | A StatusIndicator component.                                                           |
-| label               | String        | null      | No       | A label which renders next to the child contents.                                      |
-| commentCount        | String        | "0"      | No       | Displays the passed comment count                                      |
+| children            | Node          | N/A       | Yes      | 
+| stacked             | Bool          | N/A       | Yes      | Adds a stacked modifier to apply stacked style. |
+| bordered            | Bool          | N/A       | Yes      | Adds a bordered modifier to apply padding and boarders.                    |
+| *                   | N/a           | N/A       | No       | Spreads the remaining props onto the root component.                    |
```

Comment:
👍 Like this. Maybe we should specify what the root 
element is?

Reply:
I don't think we need to over document it. Check the code if you wanna know details like that (IMO).